README.md

```diff
@@ -278,12 +278,19 @@ In order to run full test suite, simply execute:
 
 **Run integration tests:**
 
-To run the integration tests, uncomment the following line from `tox.ini` and add the paths to your Kafka and Confluent Schema Registry instances. You can also run the integration tests outside of `tox` by running this command from the source root.
+To run the confluent-kafka-python integration tests you will need access to a running Kafka cluster. In order to execute successfully client will need create, read and write topics. Additionally you will need access to Zookeeper in order to set a request quota for the client `throttled-client`.
 
-    examples/integration_test.py <kafka-broker> [<test-topic>] [<schema-registry>]
+```
+kafka-configs  --zookeeper <zookeeper host >:< zookeeper port> \
```

Comment:
I dont think we want this leave of detail in the top-level README, instead add a section about integration tests (and throttle) to tests/README.md

Reply:
Remove trailing and heading spaces from `<zookeeper host >` and `< zookeeper port>`

Reply:
ack